### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -10,6 +10,7 @@
 * Dropped deprecated APIs of old makegateway names.
 * Removed ``py`` testing dependency.
 * Explicitly pass ``encoding`` when opening files in the gateway to get rid of warnings when using ``PYTHONWARNDEFAULTENCODING=1`` (#195).
+* Fixed error when loading source code files containing non-ascii characters.
```
Comment (nicoddemus) [inline CHANGELOG.rst:13]: #201

Comment (Zac-HD) [reply CHANGELOG.rst:13]: I think non-ascii source code already worked fine; the problem was that we insert the grandparent-directory into `sys.path` and if _that path_ contained non-ascii characters then we'd crash.

Comment (Zac-HD) [inline CHANGELOG.rst:13]:
```suggestion
* Fixed error when loading source code files from a path containing non-ascii characters.
```

**Review by Zac-HD — APPROVED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (nicoddemus) [reply CHANGELOG.rst:13]: Ahh thanks!

**Comment by nicoddemus** [comment]
The branch needs to be pushed to `upstream`, not my fork. 🤦‍♂️ 

Superseded by #203.

**Comment by nicoddemus** [comment]
Waiting for deployment approval from @pytest-dev/execnet-admin. 👍

**Pull request closed without merging**